### DISCUSSION TITLE
Can’t open GitHub actions from ipad

### DISCUSSION THREAD

**Discussion by gusblack-tt** [community/community#18287] [Mobile]
Hello, currently as far as I can tell there is no support for actions in the iPad app. Is this something that is being worked on?

**Comment by brianlovin** [comment]
Hey @gusblack-tt thanks for the note — actions aren't supported right now, but we are exploring the best ways to bring them into the apps. What specific workflows or use cases did you have in mind where you'd want to see/interact with Actions on mobile?
